.jules/bolt.md

```
@@ -5,3 +5,7 @@
 ## 2025-02-20 - Optimize WP REST API Payloads with _fields
 **Learning:** Default WP REST API endpoints return full objects (content, excerpts, etc.) which is wasteful for lists. Using `_fields` allows precise selection of data, reducing payload size by >80% for collections.
 **Action:** When using `wp/v2` endpoints, always specify `_fields` for the data actually needed by the component.
+
+## 2025-02-21 - Environment Limitations
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)